.github/workflows/dluhc-build-and-publish.yml

```diff
@@ -13,7 +13,7 @@ on:
       ]
 
 env:
-  VERSION: "0.1.142" # Manually increment this version when pushing to main
+  VERSION: "0.1.143" # Manually increment this version when pushing to main
```

Comment:
@Adam-W1 May need to change the Version to 144 because of this PR?
https://github.com/communitiesuk/digital-form-builder/pull/324/files